data/grc/Erbse_1961_Xenophon-Agesilaos,Hieron,LakedaimonionPoliteia,Poroi/metadata.txt

```diff
@@ -0,0 +1,48 @@
+publicationType : "chapter"
+publicationTitle : "Überlieferungsgeschichte der griechischen klassischen und hellenistischen Literatur"
+publicationDate : "1961"
+publicationPlace : "Zürich"
+publicationSeries : "Geschichte der Textüberlieferung der antiken und mittelalterlichen Literatur"
+publicationNum: "1"
+publicationStemmaNum: ""
+publicationAuthors : "Erbse, Hartmut"
+publicationPage : "268"
+publicationLink : "http://opac.regesta-imperii.de/lang_de/anzeige.php?sammelwerk=Geschichte+der+Text%C3%BCberlieferung+der+antiken+und+mittelalterlichen+Literatur"
+workTitle : "Agesilaos"
+workViaf : "184770528"
+workTitle : "Hieron"
+workViaf : "180152138663510982422"
+workTitle : "Lakedaimonion Politeia"
+workViaf : "208152138560410980341"
+workTitle : "Poroi"
+workViaf : "305126484"
+workOrigDate : "5th/4th century BCE"
+workOrigPlace : ""
+workAuthor : "Xenophon"
+workAuthorViaf : "89597697"
+workGenre : "historiography, philosophy, economics"
```

Comment:
Do we allow multiple values here ?

Reply:
Not sure. I think I'd rather not say anything in this case, so that we move forward for now